src/charm.py

```diff
@@ -221,6 +227,11 @@ def __init__(self, *args):
                 self.on.update_status,  # type: ignore
             ],
         )
+
+        self.forward_auth = ForwardAuthRequirer(
+            self, forward_auth_requirer_config=self._forward_auth_requirer_config
```

Comment:
The intention was to differentiate between forward-auth's config provider (oathkeeper) and traefik as the requirer.
Since traefik only provides app names related via ingress, perhaps `ingress_app_names` would sound better? What do you think?

Reply:
Not sure what you mean but seems fine 😅